source/develop/release-management/features/network/manageiq_ovn.html.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Integrating OVN to ManageIQ
+category: feature
+authors: Alona Kaplan
+wiki_category: Feature
+wiki_title: Integrating OVN to ManageIQ
+wiki_revision_count: 1
+wiki_last_updated: 2017-07-09
+feature_name: Integrating OVN to ManageIQ
+feature_modules: manageiq_providers_ovirt - infra_manager, network_manager, manageiq-ui-classic - network related controllers
+feature_status: In Development
+---
+
+# Integrating OVN to ManageIQ
+
+## Summary
+Discovering OVN as the network provider of oVirt and adding the ability to show, add, update and remove networks, subnets and ports directly from ovirt-provider-ovn.
+
+## Owner
+
+*   Name: [Alona Kaplan](https://github.com/AlonaKaplan)
+
+*   Email: <alkaplan@redhat.com>
+
+## Terminology
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that brings Software Defined Networking to OVS.
+* oVirt OVN provider: is a proxy that interacts with OVN. The provider has 'neutron like' API.
+* ManageIQ: is a cloud management platform that allows centralized management of various virtualization, private cloud, public cloud, containers, middleware and software defined networking technologies.
+
+## Detailed Description
+ManageIQ has an infra provider for oVirt. The feature adds automatic discovery of oVirt's network provider (OVN provider) once the infra provider is added/refreshed.
+ManageIQ users will be able to perform CRUD actions on networks, subnets and ports directly on the OVN provider without passing through oVirt.
+ManageIQ already supports automatic discovery of the OpenStack network provider (Neturon) and has all the functionality to communicate with it (CRUD actions for networks, subnets, ports and more).
+Since oVirt's OVN provider has 'neutron like' API, the purpose of the feature is to reuse the already existing ManageIQ code (fog-openstack, ManageIQ::Providers::Openstack::NetworkManager) to communicate with the oVirt's OVN provider.
+Since there are some gaps between the 'neutron like' API oVirt's OVN provider has and the desired API ManageIQ expects to get, some adjustments to the oVirt's OVN provider are done during the development process.
+
+## Requirements
+* Discovery of oVirt network provider on the infra provider add/refresh.
+* CRUD actions on the OVN provider entities (networks, subnets, ports).
```

Comment:
CRUD is important, but we should mention showing these entities as well

Reply:
r-read